library_service/urls.py

```diff
@@ -36,4 +36,5 @@
     ),
     path("api/", include("books.urls", namespace="books")),
     path("users/", include("user.urls", namespace="users")),
+    path("api/", include("borrowings.urls", namespace="borrowings")),
 ]
```

Comment:
I think, it is better to do pathes like this:
`
    path("api/books", include("books.urls", namespace="books")),
    path("api/borrowings", include("borrowings.urls", namespace="borrowings")),
    path("users/", include("user.urls", namespace="users")),
`